e2e-tests/IIoTPlatform-E2E-Tests/Standalone/A_PublishSingleNodeStandaloneTestTheory.cs

```diff
@@ -47,11 +48,26 @@ public class A_PublishSingleNodeStandaloneTestTheory {
 
         [Fact, PriorityOrder(3)]
         public void Test_StartPublishingSingleNode_Expect_Success() {
-            var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
-
+            var cts = new CancellationTokenSource(TimeSpan.FromMinutes(1));
```

Comment:
>TimeSpan.FromMinutes(1) [](start = 50, length = 23)

why a magic number here when there used to be a configurable time?

Reply:
When running in the nested edge, this call is expected to time out, and waiting 30 minutes is too much, probably I should write it as a comment